com.empesol.timetracker.yaml

```diff
@@ -0,0 +1,56 @@
+id: com.empesol.timetracker
+app-id: com.empesol.timetracker
+runtime: org.freedesktop.Platform
+runtime-version: 24.08
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+command: app.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --env=INSTALL_TYPE=flatpak
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - "/usr/lib/sdk/openjdk21/install.sh"
+  - name: notify-send
+    buildsystem: meson
+    config-opts:
+      - -Dman=false
+      - -Dgtk_doc=false
+      - -Ddocbook_docs=disabled
+    sources:
+      - type: archive
+        # curl -Ls https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.3.tar.xz | sha256sum
+        url: "https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.3.tar.xz"
+        sha256: ee8f3ef946156ad3406fdf45feedbdcd932dbd211ab4f16f75eba4f36fb2f6c0
+  - name: timetracker
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 app.sh ${FLATPAK_DEST}/bin/app.sh
+      - install -D app.jar ${FLATPAK_DEST}/bin/app.jar
+      - install -D com.empesol.timetracker.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
+      - install -D com.empesol.timetracker.desktop ${FLATPAK_DEST}/share/applications/com.empesol.timetracker.desktop
+      - install -D com.empesol.timetracker.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/com.empesol.timetracker.svg
+    sources:
+      - type: file
+        path: app.sh
+      - type: file
+        path: com.empesol.timetracker.metainfo.xml
+      - type: file
+        path: com.empesol.timetracker.desktop
+      - type: file
+        path: com.empesol.timetracker.svg
+      - type: file
+        url: https://timetracker.empesol.com/files/TimeTrackerApp-linux-x64-1.0.0-release.jar
```

Comment:
is this redistributable?

Reply:
App can be redistributed, but it's not open source.

Reply:
where is this written, I don't see a link to the license.

Reply:
I'm the author of this app, but I didn't publish license.